README.md

```diff
@@ -161,6 +161,29 @@ the "generic" system, a new empty Spack environment will be automatically create
 different Spack environment by setting the environment variable `EXCALIBUR_SPACK_ENV`
 described above.
 
+### Local usage
+
+You can use the framework to run tests on a local system without a scheduler. After spack has
+been installed following the instructions given above, it needs to be told about the software
+installed on the system. On supported HPC systems this is handled by the spack environment. By default an
+empty spack environment will be created in `spack_environments/generic/default` when you run
+ReFrame for the first time .To find locally installed
+compilers run `spack compiler find`. To find locally installed packages run `spack external find`. This
+does not necessarily find everything and some packages may have to be added manually, in particular it is 
+worth making sure that a local MPI
+installation is added to the environment. If no MPI package has been added in `~/.spack/packages.yaml`,
```

Comment:
Don't reference `~/.spack`, it may not even exist: https://github.com/ukri-excalibur/excalibur-tests/pull/90#discussion_r1100314544